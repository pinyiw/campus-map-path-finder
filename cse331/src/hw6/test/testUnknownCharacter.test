# load graph
LoadGraph g heroes.tsv

# find path, one unknown
FindPath g Java Physics

# find path, two unknown
FindPath g Chemistry Physics