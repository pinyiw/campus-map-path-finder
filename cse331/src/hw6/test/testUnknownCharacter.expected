# load graph
loaded graph g

# find path, one unknown
unknown character Physics

# find path, two unknown
unknown character Chemistry
unknown character Physics
